docs/common/versions.js

```diff
@@ -1,20 +1,17 @@
 import Package from '~/package.json';
 
-const versionDirectories = preval`
+const VERSIONS = preval`
   const { readdirSync } = require('fs');
 
   const versionsContents = readdirSync('./pages/versions', {withFileTypes: true});
+  const versionDirectories = versionsContents.filter(f => f.isDirectory()).map(f => f.name);
+  const versions = versionDirectories.filter(
+    dir => process.env.NODE_ENV != 'production' || dir != 'unversioned'
+  );
 
-  module.exports = versionsContents.filter(f => f.isDirectory()).map(f => f.name);
+  module.exports = versions;
```

Comment:
i'd probably just get rid of the versions const and do `module.exports = versionDirectories....` but nbd either way